docker/create_container.sh

```diff
@@ -3,11 +3,20 @@
 # Create the docker container with a bind mount:
 echo "Creating container..."
 . ./docker_config.sh
-execute "docker run -d \
-  --name $DOCKER_CONTAINER_NAME \
-  --mount type=bind,source=\"$(pwd)\"/../..,target=/share \
-  $DOCKER_IMAGE_NAME \
-  sleep infinity"
+if [ -n "ON_LINUX" ]; then
+  execute "docker run -d \
+    --name $DOCKER_CONTAINER_NAME \
+    --mount type=bind,source=\"$(pwd)\"/../..,target=/share \
+    --add-host=host.docker.internal:host-gateway \
```

Comment:
Let's avoid code duplications here. Instead inside the if statement, just create a variable that holds the `--add-host` option. The variable should be initialized to the empty string, so if we are not on linux, the option is nothing.